packages/overlay/src/integrations/sentry/components/traces/spans/SpanItem.tsx

```diff
@@ -94,7 +101,7 @@ const SpanItem = ({
           </span>
         </div>
         <div
-          className="waterfall"
+          className={classNames('waterfall', isQueried ? '!bg-transparent' : '')}
```

Comment:
Not happy that we had do spread `query` and `isQueried` around so much. I understand that this is kind of the result of us using `classNames` instead of creating specific CSS classes/styled components. Is there a way out of this? :D

Reply:
Yeah, that is because of tailwind CSS.
Although if needed we can pass the classnames and create the styles in .css files.